egov/egov-collectionweb/src/main/java/org/egov/collection/web/actions/receipts/BankRemittanceAction.java

```diff
@@ -219,9 +231,26 @@ public String listData() {
                 serviceCodeList.add(basm.getBusinessDetails());
             }
             final CFinancialYear financialYear = financialYearDAO.getFinancialYearById(finYearId);
+            Long dateInLongFromDate=0l;
+            Long dateInLongToDate=0l;
```

Comment:
No need of initialising these variables. Instead use them where it is being used